crates/icrate/Cargo.toml

```diff
@@ -117,6 +111,54 @@ unstable-private = []
 # https://github.com/madsmtm/objc2/issues/new
 unstable-static-nsstring = []
 
+# Examples
```

Comment:
I think I'd like these to have the prefix `unstable-` to clearly signify that they're not part of the stable contract. Or perhaps just have them placed in a different crate altogether?

Reply:
Sure, that's a good idea. I changed these to `unstable-` prefix for now. Personally, I think it's probably fine to keep them in the same crate but I don't really have a strong opinion about it either way so whatever you think is best.